cc_workspace.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+if [ -z "$1" ]; then
+    echo "You must provide an absolute path to a workspace as positional argument!"
+    echo "bash cc_workspace.sh /absolute/path/to/workspace"
+    exit 1
+fi
+
+if [[ "$1" != /* ]]; then
+    echo "You must provide an absolute path to a workspace, not a relative path:"
+    echo "$1"
+    exit 1
+fi
+
+if [ ! -d "$1" ]; then
+    echo "The provided absolute path does not exist:"
+    echo "$1"
+    exit 1
+fi
+
+if [ -z "$TARGET_ARCHITECTURE" ]; then
+    echo "Missing TARGET_ARCHITECTURE environment variables. Please run first"
+    echo "source env.sh <TARGET_ARCHITECTURE_NAME>";
+    exit 1;
+fi
```

Comment:
Is it worth checking if the sysroot path `sysroots/$TARGET_ARCHITECTURE` exists ?

Reply:
Yes, updated